config/default/core.entity_form_display.paragraph.civictheme_message.default.yml

```
@@ -0,0 +1,63 @@
+id: paragraph.civictheme_message.default
+targetEntityType: paragraph
+bundle: civictheme_message
+mode: default
+content:
+  field_c_p_background:
+    type: boolean_checkbox
+    weight: 5
+    region: content
+    settings:
+      display_label: true
+    third_party_settings: {  }
```

YAMLlint (1.37.1)

[error] too many spaces inside empty braces

(braces)